README.md

```diff
@@ -87,6 +87,7 @@ Data can easily be accessed in the form of Numpy arrays:
     frame = camera.capture(settings)
     xyz = frame.point_cloud().copy_data("xyz") # Get point coordinates as [Height,Width,3] float array
     rgba = frame.point_cloud().copy_data("rgba") # Get point colors as [Height,Width,4] uint8 array
+    bgra = frame.point_cloud().copy_data("bgra") # Get point colors as [Height,Width,4] uint8 array
```

Comment:
This is misleading because you have not actually added `frame.point_cloud().copy_data("bgra")` in this PR.

I think the right move is to keep this line, and then actually add `frame.point_cloud().copy_data("bgra")` to this PR.

Reply:
Generally I think it would be strange to only add BGRA to 2D capture but not to 3D captures.

Let me know when that is added too and I'll review the rest.